<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ele4205</class>
 <widget class="QWidget" name="Ele4205">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>251</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>213</width>
    <height>251</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ELE4205_project</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="ConfiguratioLayout" stretch="0,0,0">
     <item>
      <widget class="QWidget" name="ConfigurationWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>231</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>231</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="StatusLayout" stretch="3,1,1,1,1,3">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="StatusLayout_2" stretch="1,2,0">
            <item>
             <widget class="QLabel" name="StatusLabel">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LineStatusHolder">
              <property name="cursor">
               <cursorShape>BlankCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit#LineStatusHolder {
	 background-color : rgba(152, 152, 152, 187)
	
}</string>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="maxLength">
               <number>50</number>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2,0">
            <item>
             <widget class="QLabel" name="ResolutionLabel">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>RÃ©solution:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ResolutionComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ImageFormatLayout" stretch="1,2,0">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="FormatLabel">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>320</width>
                <height>130</height>
               </size>
              </property>
              <property name="text">
               <string>Format:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="FormatCombotBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ConfigButtonConfigLayout">
            <item>
             <widget class="QPushButton" name="ChangeConfigurationButton">
              <property name="text">
               <string>Changer configuration</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
